ingestion-beam/src/main/java/com/mozilla/telemetry/options/OutputType.java

```diff
@@ -157,12 +162,31 @@
           .withMethod(BigQueryIO.Write.Method.STREAMING_INSERTS) //
           .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_NEVER) //
           .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND).skipInvalidRows()
+          .withExtendedErrorInfo().ignoreUnknownValues()
```

Comment:
If our payload contains additional fields not in the destination table schema, this setting makes us just drop those fields rather than erroring out.